app/views/layout.erb

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+  <head>
+  <meta charset="UTF-8">
+    <title>Fwitter</title>
+    <link rel="stylesheet" href="stylesheets/bootstrap.css" type="text/css">  
```

Comment:
Just for fun I slapped Bootstrap onto my final Sinatra project, just for the index page I told myself...

...and then I couldn't stop.  I maybe spent time over 2 or 3 days making every page look decent.  But once one page looked good the others looked horrible.

I figure it's that much closer to something I can show a prospective employer, and my girlfriend was way more impressed with my project once it looked like every other Bootstrap site.
> On Apr 9, 2016, at 7:50 PM, Randall Reed <notifications@github.com> wrote:
> 
> In app/views/layout.erb <https://github.com/learn-co-students/sinatra-fwitter-group-project-v-000/pull/79#discussion_r59125817>:
> 
> > @@ -0,0 +1,13 @@
> > +<!DOCTYPE html>
> > +<html>
> > +  <head>
> > +  <meta charset="UTF-8">
> > +    <title>Fwitter</title>
> > +    <link rel="stylesheet" href="stylesheets/bootstrap.css" type="text/css">  
> Above and beyond!
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/learn-co-students/sinatra-fwitter-group-project-v-000/pull/79/files/9dfc20de18dce246a6925028c350c9316e9e3937#r59125817>